src/Truck.java

```diff
@@ -0,0 +1,23 @@
+public class Truck extends Transport {
+    private String modelName;
+    private int wheelsCount;
+
+    public Truck(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
+    }
+
+    public void update() {
+        System.out.println("Меняем покрышки " + getWheelsCount());
+        System.out.println("Проверяем двигатель");
```

Comment:
чтобы был виден полиморфизм надо указать конкретную реализацию
```suggestion
        System.out.println("Проверяем двигатель грузовика");
```